mac

```diff
@@ -146,35 +148,13 @@ brew_install_or_upgrade 'imagemagick'
 brew_install_or_upgrade 'qt'
 brew_install_or_upgrade 'hub'
 brew_install_or_upgrade 'node'
-
-brew_install_or_upgrade 'rbenv'
-brew_install_or_upgrade 'ruby-build'
-
-# shellcheck disable=SC2016
-append_to_zshrc 'eval "$(rbenv init - --no-rehash zsh)"' 1
+brew_install_or_upgrade 'vim'
```

Comment:
Do we want vim or macvim?  I use macvim, with the `--override-system-vim` option for tmux.